forge-gui/src/main/java/forge/gui/control/FControlGameEventHandler.java

```diff
@@ -451,12 +451,6 @@ public Void visit(final GameEventPlayerStatsChanged event) {
         return processCards(cards, cardsRefreshDetails);
     }
 
-    public Void visit(final GameEventSpeedChanged event) {
-        Player p = event.player;
-        processPlayer(p, livesUpdate);
-        return processEvent();
-    }
```

Comment:
@tool4ever wouldn't that stop the GUI from updating?

Reply:
Because #6982 changed it to Command Zone display updating Player stats no longer serves a purpose